test/Constraints/members.swift

```diff
@@ -590,3 +590,36 @@ do {
   throw SR_2193_Error.Boom
 } catch let e as SR_2193_Error.Boom { // expected-error {{enum element 'Boom' is not a member type of 'SR_2193_Error'}}
 }
+
+// rdar://problem/25341015
+extension Sequence {
+  func r25341015_1() -> Int {
+    return max(1, 2) // expected-error {{type member 'max' shadows global function 'max' in module 'Swift', which cannot be called using implicit 'self.'}} {{12-12=Swift.}}
```

Comment:
Sorry to keep bikeshedding this so much 😝

The message currently implies that `Swift.max` cannot be called using implicit `self.`. But that's not what the code is trying to do.

One way of solving this linguistic problem would be replacing `which` with either `and` or `but`, so it's clear that the type member is the one that can't be called with implicit `self.`.

However, it might be clearer overall if you rearrange the clauses:

> `global function 'Swift.max' is shadowed by instance member 'max', which cannot be called using implicit 'self.'`

What do you think? (Is it correct to say `instance member`? I think that's clearer than `type member`, especially when the instance member is conflicting with a static member of the same type).